packages/singleton-manager/demo/singleton-complex/node_modules/page-a/page-a.js

```diff
@@ -0,0 +1,51 @@
+import { LitElement, html, css } from 'lit-element';
+import { overlays } from 'overlays/instance.js';
+
+export class PageA extends LitElement {
+  // getInstance(sym, fallback) {
```

Comment:
related to the symbol I think, do we need to remove or is this useful code?

Reply:
removed